models/Stable-diffusion/readme.md

```diff
@@ -0,0 +1,7 @@
+ * You have to install `checkpoints` models in this folder for `Stable Diffusion`.
+ * You can find the models in [file storage][ref-fstor] or [the original link][ref-original].
+ * The original model version requires you to share informatoion with [`huggingface.co`][ref-hugface]!
```

Comment:
```suggestion
 * The original model version requires you to share information with [`huggingface.co`][ref-hugface]!
```

Reply:
Oke

Reply:
Fixed in https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/1375/commits/e35521f36684498cccd6aec4bcee50b8156fc989